packages/gasket-plugin-elastic-apm/lib/create.js

```diff
@@ -0,0 +1,15 @@
+const { devDependencies } = require('../package.json');
+
+/** @type {import('@gasket/core').HookHandler<'create'>} */
+module.exports = function create(gasket, { pkg, files }) {
+  const generatorDir = `${__dirname}/../generator`;
+
+  pkg.add('dependencies', {
```

Comment:
Do you need to add the plugin to the gasket file and as a dependency to use the action?

```
  gasketConfig.addPlugin('pluginElasticApm', name);
  pkg.add('dependencies', {
    [name]: `^${version}`
  }); 
```